cli/lsp/language_server.rs

```diff
@@ -270,7 +270,12 @@ impl LanguageServer {
         open_docs: &open_docs,
       };
       let graph = module_graph_creator
-        .create_graph_with_loader(GraphKind::All, roots.clone(), &mut loader)
+        .create_graph_with_loader(
+          GraphKind::All,
+          roots.clone(),
+          &mut loader,
+          graph_util::NpmCachingStrategy::Eager,
```

Comment:
I feel like maybe we could maybe get away with setting this as lazy in the lsp or just only install stuff found in the package.json, deno.json, or any specifiers.